pymongo/aggregation.py

```diff
@@ -40,6 +40,7 @@ def __init__(
         user_fields=None,
         result_processor=None,
         comment=None,
+        show_expanded_events=None,
```

Comment:
Since you're implementing PYTHON-3232 in this PR can you rename the title and close PYTHON-3373 as a duplicate of PYTHON-3232?